company-profiles/accor.md

```diff
@@ -0,0 +1,25 @@
+# accor
+
+## Company blurb
+
+We are Accor. We are a worldwide Augmented Hospitality leader. We are more than 230,000 hospitality experts placing people at the heart of what we do, creating new connections & emotions for our guests, nurturing real passion for service and achievement beyond limits.
+
+## Company size
+
+16,156 employees on LinkedIn
+
+## Remote status
+
+The team is remote and in person, located in France and 110 other countries.
+
+## Region
+
+France, Issy-les-Moulineaux
```

Comment:
Region is where the company hires - the section above lists 110 countries so I'd recommend using worldwide if they hire remotely in those countries